bin/local-env/install-wordpress.sh

```diff
@@ -160,3 +160,7 @@ if [ "$WEBSTORIES_DEV_MODE" != $WEBSTORIES_DEV_MODE_CURRENT ]; then
   WEBSTORIES_DEV_MODE_RESULT=$(wp config get --type=constant --format=json WEBSTORIES_DEV_MODE | tr -d '\r')
   echo -e $(status_message "WEBSTORIES_DEV_MODE: $WEBSTORIES_DEV_MODE_RESULT...")
 fi
+
+wp config set WEBSTORIES_SVG_ENABLED true --raw --type=constant --quiet
+wp media import /var/www/html/wp-content/e2e-assets/video-play.svg
+wp config delete WEBSTORIES_SVG_ENABLED
```

Comment:
I don't think we need this whole constant just for the e2e tests, and I would like to avoid adding this constant if possible.\*

From what I can see this place here is the only reason. To make it work without a constant we could do this:

```bash
wp option patch insert web_stories_experiments enableSVG 1
wp media import /var/www/html/wp-content/e2e-assets/video-play.svg
wp option patch insert web_stories_experiments enableSVG 0
```